server/repository/src/mock/invoice.rs

```diff
@@ -262,6 +262,23 @@ pub fn mock_outbound_shipment_line_a() -> InvoiceLineRow {
     })
 }
 
+pub fn mock_inbound_shipment_line_a() -> InvoiceLineRow {
```

Comment:
Food for thought: https://github.com/openmsupply/open-msupply/issues/381

Reply:
replaced with `inline_init` - though it's slightly annoying how many fields are required. would be helpful to have a base object with values for things like the price fields, which you could then override with just the relevant fields

Reply:
`inline_init` was meant to be used just with the minimum set, i.e. 
from
```rust
InvoiceLine {
                invoice_line_row: inline_init(|r: &mut InvoiceLineRow| {
                    r.id = String::from("inbound_shipment_line_a");
                    r.invoice_id = String::from("inbound_shipment_c");
                    r.item_id = String::from("item_a");
                    r.item_name = String::from("Item A");
                    r.item_code = String::from("a");
                    r.pack_size = 1;
                    r.cost_price_per_pack = 0.0;
                    r.sell_price_per_pack = 0.0;
                    r.total_before_tax = 0.0;
                    r.total_after_tax = 0.0;
                    r.r#type = InvoiceLineRowType::StockIn;
                    r.number_of_packs = 0;
                }
```
to
```
InvoiceLine {
                invoice_line_row: inline_init(|r: &mut InvoiceLineRow| {
                    r.id = String::from("inbound_shipment_line_a");
                    r.invoice_id = String::from("inbound_shipment_c");
                    r.item_id = String::from("item_a");
                    r.r#type = InvoiceLineRowType::StockIn;
                }
```

Reply:
ah. thanks! updated 😁